org.ekstep.questionunit-1.1/renderer/components/js/components.js

```diff
@@ -47,7 +47,6 @@ org.ekstep.questionunit.baseComponent = {
     }
 }
 
-org.ekstep.questionunit = org.ekstep.questionunit || {};
```

Comment:
Why did your removed this? if org.ekstep.questionunit object is not defined, then how can we define org.ekstep.questionunit.questionComponent object?

Reply:
The below line was duplicated in multiple places in the same file
`org.ekstep.questionunit = org.ekstep.questionunit || {};`
Line number - 1
Line number - 50
Line number - 117
.
I have removed Line number 50 and 117 and kept Line number 1